notebook/templates/tree.html

```diff
@@ -19,7 +19,7 @@
       <ul id="tabs" class="nav nav-tabs">
         <li class="active"><a href="#notebooks" data-toggle="tab">Files</a></li>
         <li><a href="#running" data-toggle="tab">Running</a></li>
-        <li><a href="#clusters" data-toggle="tab">Clusters</a></li>
+        <li><a href="#clusters" data-toggle="tab" {% block clusters_tab_attributes %}{% endblock %} >Clusters</a></li>
```

Comment:
Do you mean to duplicate this ability on the others or to make the same attributes appear on all three?